### PR TITLE
fix(sso): improve last step guidance in rest cas sso authentication

### PR DESCRIPTION
**Pull request by redboul**

### CODE REVIEW

**Comment by ecuriotto** [comment]
When we speak about "service url" I think we should be more explicit explaining what it is (Bonita BPM server url?) an example should be perfect.

File: md/single-sign-on-with-cas.md [hunk 1]
```diff
@@ -281,14 +284,29 @@ Take the ST response and paste it in the url of the Bonita BPM Engine login requ
...
-**Authentication to Bonita BPM Engine**
+Use a **Bonita BPM Portal** URL where the [CAS AuthenticationFilter applies](#restricted_cas_urls) for authentication to work.
+
+::: warning
+Prefer GET over POST to authenticate because experience has shown that in some server configuration, POST parameters cannot be retrieved in the CAS authentication web filter.
+:::
+
+##### **Authentication to Bonita BPM Engine** with GET
+
+The form encoded parameter URL used as service in the previous step must be use as access point because it will be sent to the CAS server to check ticket validation.
```
Comment (abirembaut) [inline md/single-sign-on-with-cas.md]: must be used

File: md/single-sign-on-with-cas.md [hunk 2]
```diff
@@ -299,5 +317,6 @@ Take the ST response and paste it in the url of the Bonita BPM Engine login requ
 You are now logged into Bonita BPM Portal and REST API calls will succeed.
 
 ::: warning
-Cookies must be enabled in REST client side for authentication to persist across REST API calls.
+Cookies must be enabled in REST client side for authentication to persist across REST API calls.  
+Therefore, calling web application root context may not work (e.g. `/bonita` by default) because session cookie seems not to be set on all web server configurations.
```
Comment (abirembaut) [inline md/single-sign-on-with-cas.md:323]: So, in this case, what needs to be done ? Not sure I understand

**Review by abirembaut — COMMENTED** [review]
see small notes

**Pull request merged**